bin/zksim.py

```diff
@@ -0,0 +1,118 @@
+import sys
+import argparse
+import os
+import sqlite3
+from importlib import util
+
+for module in ("pandas", "sklearn"):
+    if not util.find_spec(module) is not None:
+        raise ModuleNotFoundError('Missing {0}! Please run pip3 install scikit-learn pandas'.format(module))
```

Comment:
a more python-esque way to do this (similar to the rescue import) is:

```python
try:
    from sklearn.metrics.pairwise import linear_kernel
    from sklearn.feature_extraction.text import TfidfVectorizer
    import pandas as pd
except ImportError as e:
    print(f"Missing {e.name}! Please run pip3 install scikit-learn pandas")
    exit()
```

the `ImportError` is _usually_ a [`ModuleNotFoundError`](https://docs.python.org/3/library/exceptions.html#ModuleNotFoundError) like you were doing manually

Reply:
Thanks! I was looking for what the "pythonic" way to do this would be... I had a feeling what I did was a little awkward.